examples/Wkt.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2017 United States Government as represented by the Administrator of the
+ * National Aeronautics and Space Administration. All Rights Reserved.
+ */
+
+requirejs(['../src/WorldWind',
+        './LayerManager'],
+    function (ww,
+              LayerManager) {
+        "use strict";
+
+        WorldWind.Logger.setLoggingLevel(WorldWind.Logger.LEVEL_WARNING);
+
+        var wwd = new WorldWind.WorldWindow("canvasOne");
+
+        var layers = [
+            {layer: new WorldWind.BMNGLayer(), enabled: true},
+            {layer: new WorldWind.CompassLayer(), enabled: true},
+            {layer: new WorldWind.CoordinatesDisplayLayer(wwd), enabled: true},
+            {layer: new WorldWind.ViewControlsLayer(wwd), enabled: true}
+        ];
+
+        for (var l = 0; l < layers.length; l++) {
+            layers[l].layer.enabled = layers[l].enabled;
+            wwd.addLayer(layers[l].layer);
+        }
+
+        // Example showing the usage of Well Known Text collection in real life.
+        var defaultLayer = new WorldWind.RenderableLayer("Wkt Shapes");
+        new WorldWind.WktParser("" +
```

Comment:
Nice straightforward example. I like how you've shown both the simplest usage and callback usage.